Atomic/verify.py

```diff
@@ -343,6 +350,50 @@ def assemble_nvr(self, image, image_name=None):
         else:
             return nvr
 
+    def validate_manifests(self,layers):
+        """
+        Validate a system image's layers against the the associated validation manifests
+        created from those image layers on atomic pull.
+        :param layers: list of the names of the layers to validate
+        :return: None
+        """
+        missing_manifests = []
+        for layer in layers:
+            layer = layer.replace("sha256:","")
+            manifestpath = self.get_gomtree_manifest(layer)
+            if not manifestpath:
+                missing_manifests.append(layer)
+                continue
+            tmp_dir = tempfile.mkdtemp()
+            ref = os.path.join("ociimage", layer)
+            cmd = ['ostree','checkout','--union','--repo=%s' % "/ostree/repo",ref,tmp_dir]
+            r = util.subp(cmd)
+            if r.return_code != 0:
+                util.write_out(r.stderr)
+                continue
+            r = util.validate_manifest(manifestpath, img_rootfs=tmp_dir,keywords="type,uid,gid,mode,size,sha256digest")
+            util.write_out("validation output for layer %s:\n" % layer)
+            if r.return_code == 0:
+                util.write_out("\t(no changes detected)\n")
+            else:
```

Comment:
Don't output on success.  If the tool does not find anything then dont' write output.